tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -130,6 +131,13 @@ public function testEventPage(): void {
       trim($content_banner->filter('.oe-sc-event__oe-summary')->text())
     );
 
+    // Assert registration button.
+    $link = $crawler->filter('.bcl-content-banner a[target="_blank"]');
+    $this->assertCount(1, $link);
+    $button = $link->filter('button');
+    $this->assertStringContainsString('Register', $button->text());
+    $this->assertStringContainsString('calendar-check', $button->html());
+
```

Comment:
Given that we already have `$link` as a separate variable, we should also assert the href attribute.

Reply:
And we should test one version with an internal link.
The test in oe_starter_content does not count for this because we don't have the formatter there.